### DISCUSSION TITLE
Google Recommended Additional Product JSON Fields

### DISCUSSION THREAD

**Discussion by vminvestments** [garmeeh/next-seo#1325] [Ideas]
I really like the next-seo plugin. It's so much easier than writing the script JSON out. However, there are two Product JSON fields that I would like to see added. Google is alerting me of non-critical errors that could boost traffic for my clients if they were present.

Fields:
hasMerchantReturnPolicy (in offers)
shippingDetails (in offers)

Small things like these fields can give you the edge in organic search. I would love to see these added!
![search_console_pic](https://github.com/garmeeh/next-seo/assets/123517915/f3e50f1a-c190-4208-b9c1-3970136a4a2d)

**Comment by gijsbotje** [comment]
@vminvestments I added the `hasMerchantReturnPolicy` in https://github.com/garmeeh/next-seo/pull/1511. @garmeeh published it in https://github.com/garmeeh/next-seo/releases/tag/v6.8.0.
